homeassistant/components/vizio/services.yaml

```
@@ -1,9 +1,11 @@
 update_setting:
-  description: Update the value of a setting on a particular Vizio media player device.
+  name: Update the value of a setting on a Vizio media player device
```

These will be displayed in a dropdown, we should keep them short and to the point.
The description allows for longer texts.

```suggestion
  name: Update a Vizio media player setting
```